web/views.py

```
@@ -2569,41 +2581,49 @@ def checkout_success(request):
         # Process enrollments
         for item in cart.items.all():
             if item.course:
-                # Create enrollment for full course
+                # Check for an active discount coupon for this course
+                discount = Discount.objects.filter(
+                    user=user, course=item.course, used=False, valid_until__gte=timezone.now()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)